integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CarbonShowCacheCommand.scala

```diff
@@ -190,6 +192,9 @@ case class CarbonShowCacheCommand(tableIdentifier: Option[TableIdentifier],
        * Assemble result for table
        */
       val carbonTable = CarbonEnv.getCarbonTable(tableIdentifier.get)(sparkSession)
+      if (!isValidTable(carbonTable, sparkSession)) {
```

Comment:
It's better use 'Checker.validateTableExists(databaseNameOp, tableName.get, sparkSession)'.

Reply:
Done.